rclcpp/include/rclcpp/memory_strategy.hpp

```diff
@@ -101,6 +155,11 @@ class MemoryStrategy
   std::vector<rclcpp::subscription::SubscriptionBase::SharedPtr> subs;
   std::vector<rclcpp::service::ServiceBase::SharedPtr> services;
   std::vector<rclcpp::client::ClientBase::SharedPtr> clients;
+
+  std::vector<void *> subscription_handles;
```

Comment:
Yes resize is safe it will allocate if not enough is reserved. The title was about reducing allocations I didn't know if we wanted to do the preallocation. If this is just the default it will work fine.